src/Configuration/Programmer/Programmer_helper.cpp

```diff
@@ -376,6 +376,19 @@ std::string buildFpgaProgramCommand(const Cable& cable, const Device& device,
   return cmd.str() + buildInitEndStringWithCommand(programCommand.str());
 }
 
+std::string buildOTPProgramCommand(const Cable& cable, const Device& device,
```

Comment:
I think you can add unit test for this function that is similar to `buildFpgaProgramCommand` in tests/unittest/CFGProgrammer/CFGProgrammer_test.cpp file